src/player.cpp

```diff
@@ -634,7 +634,6 @@ PlayerErrors Player::play(
 
 void Player::stop(unsigned int handle)
 {
-    removeHandle(handle);
```

Comment:
[nit] I guess I'd like to see it documented (in comments in this file, ideally) who is responsible for removing the handle (e.g. calling `Player::removeHandle()`).